src/main/java/yesman/epicfight/api/client/camera/EpicFightCameraAPI.java

```diff
@@ -896,11 +896,17 @@ public void fireCameraBuildPost(Camera camera, float partialTick) {
 	}
 	
 	/**
-	 * REturns a new basis for {@link LivingEntity#yRotHead} instead of coupling it to {@link Entity#getYRot}
+	 * Returns a new basis for {@link LivingEntity#yRotHead} instead of coupling it to {@link Entity#getYRot}
 	 */
```

Comment:
Just one more change:

```suggestion
/// Returns a new basis for [LivingEntity#yRotHead] instead of coupling it to [Entity#getYRot].
///
/// This method takes a [Player] instead of [LocalPlayer] because casting
/// to the client-only [LocalPlayer] inside a mixin (e.g., in [LivingEntity])
/// would crash a dedicated server due to Forge's `@OnlyIn(Dist.CLIENT)`.
```